packages/di/src/common/services/InjectorService.ts

```
@@ -234,20 +188,17 @@
           return instance;
         });
-        // locals?.delete(DI_USE_PARAM_OPTIONS);
         return instance;
 
       case ProviderScope.REQUEST:
-        if (locals) {
-          locals.set(token, instance);
+        if (options.locals) {
+          options.locals.set(token, instance);
 
           if (provider.hooks && provider.hooks.$onDestroy) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)